tools/release-tests/mirror-json-test.py

```diff
@@ -16,12 +16,15 @@
 parser = argparse.ArgumentParser(description='Download and test mobilecoind / mirror and json gateway')
 parser.add_argument('entropy', metavar='e', type=str, help='Root entropy of account for testing')
 parser.add_argument('--url', metavar='u', type=str, help='URL of release tarball', default='https://github.com/mobilecoinofficial/mobilecoin/releases/latest/download/mobilecoind-mirror-tls.tar.gz')
+parser.add_argument('--synced', action='store_true', help='Do not delete ledger-db and mobilecoind-db on start')
```

Comment:
I've been making reuse the default and using `--clean` for this elsewhere

Reply:
Or perhaps `--skip-clean`. I think `--synced` does not describe what this actually does.

Reply:
`--skip-clean` works great - fixed!